test/suites/one-liner-masking.yml

```diff
@@ -0,0 +1,32 @@
+name: one-liner features
+testcases:
+- name: one-liner
+  steps:
+  - script: |-
+      echo '{"value": ""}' | pimo --mask "value=[{fluxUri: 'pimo://nameFR'}]"
```

Comment:
If `pimo --mask "value={fluxUri: 'pimo://nameFR'}"` is implemented, can you add a test for it ?